cglib/src/main/java/net/sf/cglib/core/KeyFactory.java

```diff
@@ -84,14 +84,30 @@
         938313161, 1288102441, 1768288259  };
     
 
-    public static final Customizer CLASS_BY_NAME = new Customizer() {
+    public static final Customizer CLASS_BY_NAME = new Customizer2() {
         public void customize(CodeEmitter e, Type type) {
```

Comment:
Technically speaking, if someone uses CLASS_BY_NAME as Customizer, he/she would expect `Customizer.customize(CodeEmitter e, Type type)` do some meaningful stuff.

So, the implementation is there for backward compatibility